scripts/imagebuild.sh

```diff
@@ -0,0 +1,56 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+root_dir="$(cd "$(dirname "$0")/.." ; pwd)"
+base_dir="${root_dir}"
+dockerfile_dir="${base_dir}/images/federation-v2"
+cd "$base_dir" || {
+  echo "Cannot cd to '$base_dir'. Aborting." >&2
+  exit 1
+
+}
+cd ${base_dir}
+
```

Comment:
we should do a check here to ensure that we bail early if there's no work to do - for example during a PR build